support-figma/extended-layout-plugin/src/data/shader/gradient_flow.txt

```diff
@@ -0,0 +1,75 @@
+// Source: https://www.shadertoy.com/view/wdyczG
+float S(float a, float b, float t){
+  return smoothstep(a,b,t);
+}
+
+mat2 Rot(float a)
+{
+    float s = sin(a);
+    float c = cos(a);
+    return mat2(c, -s, s, c);
+}
+
+
+// Created by inigo quilez - iq/2014
+// License Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License.
```

Comment:
We can't use code with this license, according to [go/licensetable](https://goto.google.com/licensetable) it's forbidden.

Reply:
Yeah - I am guessing (we should check) that the shaders included in the skia shaders playground are OK for us to include here. It's hard to find any license info for stuff from shadertoy... since these are just samples of what can be done, it's OK to have fewer of them.

Reply:
Removed most of them.

There is one `mesh2d.txt` I copied from the DisplaySafety demo which is `inspired from` a shader toy shader. Also a `road.txt` which is similarly modified from a shader toy shader. Is it ok to keep them?

Reply:
I removed those two modified shader toy shaders.